clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_10_0.xsd

```diff
@@ -0,0 +1,1499 @@
+                    after which a java.util.NoSuchElementException will be thrown.
+                    If undefined, the pool will block indefinitely.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="min-evictable-idle-time" type="xs:long" default="1800000">
+            <xs:annotation>
+                <xs:documentation>
+                    Specifies the minimum amount of time that an connection may sit idle in the pool before it is eligible for eviction due to idle time.
+                    When non-positive, no connection will be dropped from the pool due to idle time alone.
+                    This setting has no effect unless timeBetweenEvictionRunsMillis > 0.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="min-idle" type="xs:int" default="1">
+            <xs:annotation>
+                <xs:documentation>
+                    Sets a target value for the minimum number of idle connections (per server) that should always be available.
+                    If this parameter is set to a positive number and timeBetweenEvictionRunsMillis > 0, each time the idle connection eviction thread runs,
+                    it will try to create enough idle instances so that there will be minIdle idle instances available for each server.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:simpleType name="exhausted-action">
+        <xs:restriction base="xs:token">
+            <xs:enumeration value="EXCEPTION">
+                <xs:annotation>
+                    <xs:documentation>
+                        An exception will be thrown to the calling user.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="WAIT">
+                <xs:annotation>
+                    <xs:documentation>
+                        The caller will block by invoking waits until a new or idle connections is available.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="CREATE_NEW">
+                <xs:annotation>
+                    <xs:documentation>
+                        A new persistent connection will be created and returned thus rendering maxActive meaningless.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:simpleType name="protocol-version">
+        <xs:restriction base="xs:token">
+            <xs:enumeration value="1.0"/>
+            <xs:enumeration value="1.1"/>
+            <xs:enumeration value="1.2"/>
+            <xs:enumeration value="1.3"/>
```

Comment:
Versions prior to 2.0 have been dropped:

https://github.com/infinispan/infinispan/blob/10.1.7.Final/client/hotrod-client/src/main/java/org/infinispan/client/hotrod/ProtocolVersion.java#L27

Reply:
Good catch.